scripts/inclusion-bot.js

```diff
@@ -0,0 +1,53 @@
+const utils = require("../utils");
+
+const bots = [
+  {
+    // "guys bot"
+    hear: /\bguy[sz]\b/,
+    regex: /(?<!boba )(?<!five )(?<!5 )(?<!halal )guy(s|z)(?=[^"“”']*(["“”'][^"“”']*["“”'][^"“”']*)*$)/i,
+    alternatives: ["y'all"],
+  },
+  {
+    hear: /\b(lame)\b/,
```

Comment:
we should probably allow "lame duck" or else presidential transitions are going to be really awful every four years.